### ISSUE TITLE
Steam review scores: Valve doesn't know best

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#677]
##The tl;dr version

Instead of showing the "headline" score from the top of the page:

![steam-reviewscore.PNG](https://bitbucket.org/repo/b6qGGA/images/1630070003-steam-reviewscore.PNG),

you should divide the Positive total (in parentheses beside the radiobutton) by the All total

![steam-reviewscore3.PNG](https://bitbucket.org/repo/b6qGGA/images/585217316-steam-reviewscore3.PNG)  

in this case giving 18 / 27 = 67%, which is quite different from, and less misleading than 92%! (this example is [Rescue your chickens](http://store.steampowered.com/app/551080/))

I suggest using the radiobutton captions because they are part of the static HTML, not loaded dynamically by JS.

It would also help a lot to show the number of reviews the score is based on (in this case the 27 total) beside the percentage, because even the total here might be a small number, which makes the score more uncertain!

##The long version

How often do you see this:

![hundred-percent.PNG](https://bitbucket.org/repo/b6qGGA/images/2581733355-hundred-percent.PNG)

Do you think "Wow, five perfect games, I gotta buy this bundle!" Of course not (well some users would), you think "Sigh, more unpopular games with very few reviews, I'll have to carefully investigate each one". Even if you haven't studied statistics, you know what's going on. Fewer reviews means less confidence in the result.

Valve unfortunately made the problem worse. Some developers were gaming the system by giving out free keys in exchange for positive reviews. This meant that the "Other" category would have a significantly higher score than the "Steam purchasers" category. Valve's solution was to create those categories, and make the "headline" score only count the latter category. Not only does this mean fewer reviews to contribute to the average, but they introduced a "bug" (or was it deliberate?) which meant customers who were so disgusted with the game that they refunded it would have their reviews put in the "Other" category even though they had originally paid for the game. This stripped the "Steam purchasers" category of a lot of negative reviews, and meant the "Other" category would actually have the lower score. Valve claims to have fixed this bug, but judging by the number of times I've seen "Other" category scores that were much lower than "Steam purchasers", I'm thinking they didn't fix it retroactively.

Since either of those two distortions might be in place, you'd ideally take the *lower* score from the two categories, but that is more complicated, and we're also trying to avoid showing a score from a small number of reviews, so using the total (paid + other) is a good compromise, and avoids having to make extra HTTP requests.

I haven't mentioned "Your Languages" vs "All Languages". You can't cater individually to everyone, so just make it "All Languages", which the above totals give you anyway.

[Issue created by bb34887: 2017-04-09]
[Last updated on bitbucket: 2017-04-10]

[Comment created by bb34887: 2017-04-10]
Of course Valve's "headline score" is equivalent to selecting All / Steam Purchasers / All Languages. Here's [The Haunting of Billy](http://store.steampowered.com/app/445170/), with "100% positive" reviews:

![HOB.PNG](https://bitbucket.org/repo/b6qGGA/images/1000050787-HOB.PNG)

Looks like all the negative reviews got shoved into "Other" (where they don't count to the headline score) after refunding:

![HOB-other.PNG](https://bitbucket.org/repo/b6qGGA/images/1000034869-HOB-other.PNG)

[Comment created by bb34887: 2017-04-09]
I forgot to mention: the suggested procedure is equivalent to selecting all the top radiobuttons and letting the JS do its thing (though in the original description I rounded up to 67% instead of down to 66% like Valve)

![steam-reviewscore4.PNG](https://bitbucket.org/repo/b6qGGA/images/1397685057-steam-reviewscore4.PNG)
